backend/app/routers/datasets.py

```diff
@@ -522,6 +530,12 @@ async def save_files(
                         status_code=404, detail=f"Folder {folder_id} not found"
                     )
 
+            public = False
+            authenticated = False
+            if dataset.status == "PUBLIC":
+                public = True
+            if dataset.status == "AUTHENTICATED":
```

Comment:
Could you help me understand the concept of "AUTHENTICATED"?

Reply:
If a dataset is "public" then anyone can see it, even if they are not logged in.

If a dataset is "authenticated" then anyone with an account who is logged in can see it.

Reply:
thanks for the explanation!